alacritty/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "alacritty"
+version = "0.3.2"
+authors = ["Joe Wilm <joe@jwilm.com>"]
+license = "Apache-2.0"
+description = "GPU-accelerated terminal emulator"
+readme = "../README.md"
+homepage = "https://github.com/jwilm/alacritty"
+edition = "2018"
+
+[dependencies]
+alacritty_terminal = { path = "../alacritty_terminal" }
+log = "0.4"
+time = "0.1.40"
+env_logger = "0.6.0"
+crossbeam-channel = "0.3.8"
+
+[target.'cfg(target_os = "macos")'.dependencies]
+dirs = "1.0.2"
+
+[package.metadata.deb]
+maintainer = "Joe Wilm <joe@jwilm.com>"
+license-file = ["LICENSE-APACHE", "3"]
+extended-description = """\
+Alacritty is the fastest terminal emulator in existence. Using the GPU for \
+rendering enables optimizations that simply aren't possible without it.  """
+depends = "$auto"
+section = "rust"
+priority = "optional"
+assets = [
+    ["target/release/alacritty", "usr/bin/", "755"],
+    ["extra/linux/alacritty.desktop", "usr/share/applications/", "644"],
+    ["extra/logo/alacritty-term.svg", "usr/share/pixmaps/Alacritty.svg", "644"],
+    ["extra/completions/alacritty.bash", "usr/share/bash-completion/completions/alacritty", "644"],
+    ["extra/completions/alacritty.fish", "usr/share/fish/completions/alacritty.fish", "644"],
+    ["extra/completions/_alacritty", "usr/share/zsh/vendor-completions/_alacritty", "644"],
+    ["extra/alacritty.info", "usr/share/terminfo/a/alacritty", "644"],
+]
+maintainer-scripts = "extra/linux/debian"
```

Comment:
These paths all need to be changed. See https://github.com/chrisduerr/alacritty/blob/alacritty-split/alacritty/Cargo.toml#L41-L50 and the live shader reload feature needs to get passed through.

I really recommend taking a look at the Cargo.tomls from my branch, these have been tested to work with all features, tests and tools.